ConsumerApi.Tests.Integration/StepDefinitions/ChallengesApiStepDefinitions.cs

```diff
@@ -31,7 +30,7 @@ public ChallengesApiStepDefinitions(IOptions<HttpConfiguration> httpConfiguratio
     [Given("the user is authenticated")]
     public async Task GivenTheUserIsAuthenticated()
     {
-        _sdk = await Client.CreateForNewIdentity(_httpClient, _clientCredentials, PasswordHelper.GeneratePassword(20, 26));
+        _sdk = await Client.CreateForNewIdentity(_httpClient, _clientCredentials, "somePassword");
```

Comment:
I think it makes sense to have the same password for all identities across all step definitions.

=> introduce a globally available constant